ci/test/actions-demo

```diff
@@ -403,4 +403,36 @@ EOF
 
 popper run --runtime docker
 
+# Test implementation of handling exit code 78
+cat <<EOF > main.workflow
+workflow "sample" {
+  resolves = "end"
+}
+
+action "a" {
+  uses = "docker://debian:buster-slim"
+  args = "whoami"
+}
+
+action "b" {
+  uses = "JayjeetAtGithub/test-exit-code-78@master"
+  args = "whoami"
+}
+
+action "c" {
+  uses = "actions/bin/sh@master"
+  args = "whoami"
+}
+
+action "end" {
+  needs = ["a", "b", "c"]
+  uses = "sh"
+  args = "whoami"
+}
+EOF
```

Comment:
this can be simplified if we remove action `a` and `c`. Or is there anything in particular you would like to test with those two?

Reply:
Yeah We can remove them. I actually included them while doing local manual tests.